packages/labs/src/mock_io_bazel_rules_closure.bzl

```diff
@@ -0,0 +1,28 @@
+"Mock transitive toolchain dependencies that are uneeded"
```

Comment:
nice, should we doc this for users too?

Reply:
What would you recommend as far as docs?